crates/burn-train/src/logger/file.rs

```diff
@@ -16,14 +16,20 @@ impl FileLogger {
     /// # Returns
     ///
     /// The file logger.
-    pub fn new(path: &str) -> Self {
+    pub fn new(path: &Path) -> Self {
```

Comment:
Shouldn't this be used as `impl AsRef<Path>`

Reply:
It should. I am updating the branch.